agpypeline/checkmd.py

```diff
@@ -0,0 +1,22 @@
+"""CheckMD Class"""
+
+from typing import List, NamedTuple, Optional, TextIO
+
+
+# pylint: disable=invalid-name
+class CheckMD(NamedTuple):
+    """This is the CheckMD class based off of NamedTuple which can be used in
+    order to store data passed into argparse."""
+    timestamp: str
+    season: str
+    experiment: str
+    working_folder: str
+    list_files: List[TextIO]
+    container_name: Optional[str] = None
+    target_container_name: Optional[str] = None
+    trigger_name: Optional[str] = None
+    context_md: Optional[str] = None
+
+    def files(self):
```

Comment:
files() was changed to get_list_files() and a new function get_working_folder() was added in order to work with soilmask.py.

Reply:
@jvanderleeuw Is this incorrect in this branch?